### ISSUE TITLE
Tachiyomi makes my phone lag and overheat A LOT

### ISSUE THREAD

**Issue by Elakiim** [tachiyomiorg/tachiyomi#6048]
### Steps to reproduce

Example:
1.  I update my library
2.  The manga process starts normally
3.  Around half of what is my full library the phone starts lagging and it becomes super hot
4. When the update finishes the phone goes back to normal 

### Expected behavior

Well it shouldn't lag so much and overheat 

### Actual behavior

It lags, overheats and consequentially i can't read other manga, or use any other app on my phone because it lags too much.

### Crash logs

No crashes

### Tachiyomi version

0.12.3

### Android version

Android 11 RP1A.200720.011

### Device

Xiaomi Redmi Note 8 Pro

### Other details

So far it's been something like 2-3 updates that this thing keeps happening, before it never happened.
It is strange because other apps/games like, for example, Dead Cells or Bleach Brave Souls don't lag, not even a bit and my phone doesn't gets hot, it happens only as said, when i update my library on Tachiyomi.
.
.
.
.
Also i'm new on Gifthub, so i'm sorry if there is something wrong, but i tried to get help on the discord and there they suggested me to make a "post" here.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.12.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by CarlosEsco** [comment]
https://github.com/tachiyomiorg/tachiyomi/issues/5702

**Comment by Elakiim** [comment]
> #5702

That is not what happens to me, my battery isn't drained by the app. Also when i read any manga the app works well, the problem starts when i update my library and ends right after it finished updating it.

Also this isn't a problem related to the version 0.12.3 because, as said, it happened also in the past with previous versions.

**Comment by CarlosEsco** [comment]
How many manga are you trying to update when this happens?

**Comment by Elakiim** [comment]
> How many manga are you trying to update when this happens?

about 450/500

**Comment by arkon** [comment]
It's probably the same thing as #5702. The users experiencing the symptoms haven't been able to pinpoint what's causing it specifically, but some think it's the library updates too.

**Comment by FourTOne5** [comment]
Im not quite sure if it's related but when tachi does libary update if the connection is fast the update of manga finishes quickly and after like 30-40 manga tachi starts to lag and if you disconnect the Internet tachi stops responding for a second. Maybe adding a little delay before updating the next manga in queue may fix this.

**Comment by arkon** [comment]
Better to add on to #5702 than have another issue.

**Comment by Elakiim** [comment]
> Im not quite sure if it's related but when tachi does libary update if the connection is fast the update of manga finishes quickly and after like 30-40 manga tachi starts to lag and if you disconnect the Internet tachi stops responding for a second. Maybe adding a little delay before updating the next manga in queue may fix this.

No, for me that doesn't happen. While it's true that the first 30-40 manga are updated quickly and after those it starts lagging and overheating regardless of my connection (which is always "fast").
